appinfo/routes.php

```diff
@@ -66,6 +66,11 @@
 ['name' => 'folder_api_v2#update', 'url' => '/api/v2/folders/{folderId}', 'verb' => 'PATCH'],
 ['name' => 'folder_api_v2#delete', 'url' => '/api/v2/folders/{folderId}', 'verb' => 'DELETE'],
 
+// API 1.3
```

Comment:
Actually this was something I just wanted to ask here, since I was thinking about it yesterday.. :slightly_smiling_face: Would also be the cleaner version in my opinion.